quickwit-directories/src/lib.rs

```diff
@@ -27,14 +27,17 @@ This crate contains all of the building pieces that make quickwit's IO possible.
 - The `HotDirectory` wraps another directory with a static cache.
 - The `CachingDirectory` wraps a Directory with a dynamic cache.
 - The `DebugDirectory` acts as a proxy to another directory to instrument it and record all of its IO.
+- The `BundleStorage` bundles together multiple files into a single file.
 */
 #![warn(missing_docs)]
 
+mod bundle_storage;
```

Comment:
I would put bundle_storage into quickwit-storage as here we only keep structs that implement `Directory` trait.

Reply:
I moved it